static/src/javascripts-legacy/projects/common/modules/experiments/tests/membership-engagement-banner-tests.js

```diff
@@ -27,7 +27,7 @@ define([
         this.author = 'Roberto Tyley';
         this.description = 'Show contributions/membership messages for the ' + edition + ' edition.';
         this.showForSensitive = false;
-        this.audience = 0.5;
+        this.audience = 1.0;
```

Comment:
No, but only because there are no engagement banner tests currently being supported. If the `EditionTest` function is intended to be used to implement multiple tests simultaneously (this is my understanding), then I think `audience` and `audienceOffset` should be function parameters. But I guess that's outside the scope of this pull request.